src/string_scanner.cr

```diff
@@ -280,6 +282,37 @@ class StringScanner
     @str.byte_slice(@byte_offset, @str.bytesize - @byte_offset)
   end
 
+  # Returns one byte from the current offset.
+  # ```
+  # require "string_scanner"
+  #
+  # s = StringScanner.new("あ")
+  # s.read_byte # => 227
+  # s.read_byte # => 129
+  # s.read_byte # => 130
+  # ```
+  def read_byte : UInt8?
+    return nil if eos?
+    s = @str.byte_at(@byte_offset)
+    @byte_offset += 1
+    s
```

Comment:
Avoid using one letter variable names, please.

```suggestion
    byte = @str.byte_at(@byte_offset)
    @byte_offset += 1
    byte
```

Reply:
OK, I corrected it.